profile_bluesky/startup/instrument/devices/ad_rigaku_detector.py

```diff
@@ -171,14 +162,10 @@ def staging_setup_DM(self, *args, mode=None):
             self.stage_sigs["imm1.file_name"] = "test"
             # TODO: what else is needed?
 
-        self.stage_sigs["cam.file_path"] = os.path.dirname(f"{fname}.bin")
-        self.stage_sigs["cam.file_name"] = os.path.basename(f"{fname}.bin")
-
     @property
     def images_received(self):
         """
         Return the number (int) of images captured.
-
         suggestion:  ``self.immout.num_captured.get()``
         """
         raise NotImplementedError("Must implement in detector-specific subclass.")
```

Comment:
Is there work to do here?  Won't this exception be a problem when we try to use it?